plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -1136,14 +1376,25 @@ public static function get_tables() {
 			wc_get_container()->get( DataRegenerator::class )->get_lookup_table_name(),
 		);
 
+		$wca_tables = array(
```

Comment:
Looks like we don't use them separately. Could we just merge `$wca_tables` into `$tables`?